verl/third_party/vllm/vllm_v_0_5_4/tokenizer.py

```diff
@@ -70,7 +70,7 @@ def get_lora_tokenizer(self, lora_request: Optional[LoRARequest]) -> "PreTrained
     # FIXME(sgm): for simplicity, we assign the special token here
     @property
     def pad_token_id(self):
-        return self.tokenizer.pad_token_id
+        return self.tokenizer.pad_token_id if self.tokenizer.pad_token_id is not None else self.tokenizer.eos_token_id
```

Comment:
Actually, we can simply set pad_token_id to eos_token_id after tokenizer initializes in the trainer. This makes code changes significantly fewer.